content/en/continuous_testing/environments/multiple_env.md

```diff
@@ -136,9 +131,6 @@ With this override, the URL `https://my-app.com/some/path` gets rewritten as `ht
 The <code>resourceUrlSubstitutionRegexes</code> is also applied to the first request, similarly to <code>startUrl</code> and <code>startUrlSubstitutionRegex</code>.
 </div>
 
-<div class="alert alert-info">
```

Comment:
same suggestion as above (don't forget to change `startUrlSubstitutionRegex` to `resourceUrlSubstitutionRegexes`)